arviz/plots/backends/bokeh/posteriorplot.py

```diff
@@ -187,6 +187,15 @@ def display_rope(max_data):
             line_color="red",
             line_alpha=0.7,
         )
+        less_than_left_probability = (values < vals[0]).mean()
+        less_than_right_probability = (values <= vals[1]).mean()
+        probability_within_rope = less_than_right_probability - less_than_left_probability
+        ax.text(
+            x=(vals[0]+vals[1])/2,
+            y=[max_data * 0.5],
```

Comment:
Can you try other values and see if you can get the rope text to be between the ref_val and hdi text?